doc/classes/VisualInstance3D.xml

```diff
@@ -63,6 +63,7 @@
 		</member>
 		<member name="sorting_offset" type="float" setter="set_sorting_offset" getter="get_sorting_offset" default="0.0">
 			The sorting offset used by this [VisualInstance3D]. Adjusting it to a higher value will make the [VisualInstance3D] reliably draw on top of other [VisualInstance3D]s that are otherwise positioned at the same spot.
+			Since this offset is based on the same measuring unit as the game, its value should be at least slightly higher than one unit in order for its effect to be visible.
```

Comment:
This isn't quite accurate. In https://github.com/godotengine/godot/issues/73947 it needed to be a little bit higher than 1 only because the two decals were a little more than one unit apart. 

I would say something like: "

```suggestion
			The amount by which the depth of this [VisualInstance3D] will be adjusted when sorting by depth.  Uses the same units as the engine (which are typically meters). Adjusting it to a higher value will make the [VisualInstance3D] reliably draw on top of other [VisualInstance3D]s that are otherwise positioned at the same spot. To ensure it always draws on top of other objects around it (not positioned at the same spot), set the value to be greater than the distance between this [VisualInstance3D] and the other nearby [VisualInstance3D]s.
			
```

Reply:
Makes sense. I'll make a new commit.
Thx